base/database/testing.go

```diff
@@ -2,11 +2,91 @@ package database
 
 import (
 	"app/base/models"
+	"app/base/utils"
 	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
 )
 
+func DebugWithCachesCheck(part string, fun func()) {
+	fun()
+	validAfter, err := CheckCachesValid()
+	if err != nil {
+		utils.Log("error", err).Error("Could not check validity of caches")
+	}
+
+	if !validAfter {
+		utils.Log("part", part).Panic("Cache mismatch created")
+	}
+}
+
+// nolint: funlen
```

Comment:
This lint is unneccessarry. It's better to split method into multiple submethods. That's why we use the lint :wink:.